Source/Features/Store/NotificationStore.swift

```diff
@@ -506,4 +515,15 @@ public class NotificationStore: Collection, StoreRealTimeObserver {
             setUnseenCount(unseenCount - 1, notifyObservers: true)
         }
     }
+
+    private var publisherInstance: AnyObject?
+    @available(iOS 13.0, *)
+    public func publishera() -> NotificationStorePublisher {
```

Comment:
@jfilowk this method must be renamed (publishera is just invalid).

If we do it as a singleton, then probably is better if we make it a `public var publishers: NotificaitonStorePublisher { // Implemnetation }`.

I'd call it `publishers` because it contains all publishers for all attributes. Maybe the class should be called `NotircatinStorePublishers` as well (plural).

Reply:
Actually not, I've just removed this.